source/includes/api/_dashboard.md

```diff
@@ -209,13 +209,14 @@ description     | Description of the dashboard
 content         | Content of the dashboard, typically encoded as a JSON string                 | any valid text
 published       | If the dashboard is in a publishable state                                   | boolean
 photo           | Object containing a set of image urls associated with the dashboard          | object
-user_id         | Id of the user who created the dashboard                                     | string with valid user id (not validated)
+user_id         | Id of the user who created the dashboard. This field is automatically set using the id of the user who performed the request. | -
```

Comment:
IMO this is confusing. If the `user_id` value passed on POST is ignored, then it shouldn't be in this list, it should only show up on the GET documentation.

Reply:
Agreed 👍